src/server/scripts/Northrend/zone_howling_fjord.cpp

```diff
@@ -256,15 +294,27 @@ class spell_fjord_mindless_abomination_explosion_fx_master : public SpellScript
     void HandleScript(SpellEffIndex /*eff*/)
     {
         Unit* caster = GetCaster();
-
-        for (uint8 i = 0; i < 11; ++i)
-            caster->CastSpell(caster, SPELL_RANDOM_CIRCUMFERENCE_POINT_POISON);
-
-        for (uint8 i = 0; i < 6; ++i)
-            caster->CastSpell(caster, SPELL_RANDOM_CIRCUMFERENCE_POINT_BONE);
-
-        for (uint8 i = 0; i < 4; ++i)
-            caster->CastSpell(caster, SPELL_RANDOM_CIRCUMFERENCE_POINT_BONE_2);
+        caster->CastSpell(caster, SPELL_RANDOM_CIRCUMFERENCE_POINT_POISON);
```

Comment:
isn't it best to create a constexpr array with all of these and just loop it which keeps the order you're using?

Reply:
Imo that is making it overcomplicated